executionserver/image.yaml

```diff
@@ -51,6 +51,9 @@ envs:
     - name: "KIE_SERVER_BYPASS_AUTH_USER"
       example: "false"
       description: "KIE execution server bypass auth user (Sets the org.kie.server.bypass.auth.user system property)"
+    - name: "KIE_SERVER_SYNC_STARTUP"
+      example: "false"
+      description: "KIE execution server sync startup with containers (the server will hold the startup until containers are loaded), only for managed mode (Sets the org.kie.server.sync.deploy system property)"
```

Comment:
Actually it can be applied even for unmanaged more. In unmanaged mode the Kie server use configuration stored in local data storage to configure containers, but the property is still applied when starting the Kie server - Kie server isn't started until all containers are initialized.